README.md

```diff
@@ -249,6 +231,169 @@ class UserController extends Controller {
 }
 ```
 
+## Advanced Usage
+If you have an API that doesn't follow the conventions of this driver you need to configure some more things.
+We have defined some entity annotations which you can use to define target URLs.
+
+```php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as REST;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @REST\Get("http://www.yourSite.com/api/products/findOne/{id}")
```

Comment:
ob dann get der richtige name is, weiß ich auch nich: Was ich sagen will is: so is es nichtssagen, weil kien zusammenhang zu dem mapping hergestellt wird semantisch